ethereum/circuits/get_receipt/src/main.nr

```diff
@@ -1,14 +1,5 @@
-use dep::ethereum::receipt::{get_receipt, TxReceiptWithinBlock, PhantomReceiptProofLen, PhantomReceiptRlpLen};
+use dep::ethereum::receipt::{get_receipt, TxReceiptWithinBlock};
 
 fn main(chain_id: pub Field, block_number: pub u64, tx_idx: pub Field) -> pub TxReceiptWithinBlock {
-    let phantom_receipt_proof_len: PhantomReceiptProofLen <3724>  = PhantomReceiptProofLen {}; // = MAX_RECEIPT_TREE_DEPTH * MAX_TRIE_NODE_LENGTH
-    let phantom_receipt_rlp_len: PhantomReceiptRlpLen <525>  = PhantomReceiptRlpLen {};
-
-    get_receipt(
-        chain_id,
-        block_number,
-        tx_idx,
-        phantom_receipt_proof_len,
-        phantom_receipt_rlp_len
-    )
+    get_receipt(chain_id, block_number, tx_idx)
```

Comment:
Those phantoms should still be here to show that the library supports different lengths

Reply:
Showed offline that they can't be here